assignments/arrays.js

```diff
@@ -62,35 +64,76 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// The dealer can't recall the information for a car with an id of 33 on his lot.
+//Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
+
+console.log(`Car 33 is a ` + inventory[32].car_year + ' ' + inventory[32].car_make + ' ' + inventory[32].car_model);
 
 
 
 // ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?
+//Log the make and model into the console.
+
+
 let lastCar = 0;
-console.log();
+lastCar = inventory.length - 1;
+console.log(inventory[lastCar].car_make + ' ' + inventory[lastCar].car_model);
 
 // ==== Challenge 3 ====
-// The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
+// The marketing team wants the car models listed alphabetically on the website.
+//Sort all the car model names into alphabetical order and log the results in the console
+
+
 let carModels = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i++){
+  carModels.push(inventory[i].car_model);
+  carModels.sort();
```

Comment:
Consider normalizing the case. In this dataset there are both Lower and Upper casing.

Try to think about what would have to be done to sort everything with that in mind.